sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,13 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
-    sT(Float('1.23456789', prec=9), "Float('1.23456788994', prec=9)")
-    sT(Float('1.234567890123456789', prec=19),
-       "Float('1.234567890123456789013', prec=19)")
-    sT(Float(
-        '0.60038617995049726', 15), "Float('0.60038617995049726', prec=15)")
+    sT(Float('1.23', dps=3), "Float('1.22998046875', prec=13)")
+    sT(Float('1.23456789', prec=33),
+            "Float('1.23456788994371891021728515625', prec=33)")
+    sT(Float('1.234567890123456789', prec=66),
+       "Float('1.23456789012345678901321800735590983322254032827913761138916015625', prec=66)")
+    sT(Float('0.60038617995049726', 15),
+       "Float('0.6003861799504972562857574303052388131618499755859375', prec=53)")
```

Comment:
Yes , the `_print_Float()` in `repr.py` uses the `to_str` function of mpmath. The `to_str` function uses decimal precision to yield the string representation of a mpmath float. 

I'll make the required changes.